README.md

```diff
@@ -23,3 +23,20 @@ More information about Ancona game engine can be found at the [Ancona github pag
 ### Running Unit Tests
 
 `npm test`
+
+## Configuration Options
+
+Duckling has an optional static configuration file. The file is located at `$HOME/.duckling/options.json`.
+
+### WebGL vs Canvas Renderer
+
+Ducking defaults to WebGL for rendering the game map. Some graphics drivers have problems
+with the WebGL renderer. You can use the canvas renderer by setting the "useWebGL" key to false.
+
+#### Example
+`
+// In $HOME/.duckling/options.json
```

Comment:
I wouldn't reference the "In file" here, you mentioned where the file is above. Also, the { } aren't rendered on their own line